src/TwigComponent/tests/Fixtures/templates/components/UserCard.html.twig

```diff
@@ -0,0 +1,6 @@
+{% props user %}
+
+<div {{ attributes }}>
+    <p>{{ user.name }}</p>
+    <p>{{ user.email }}</p>
```

Comment:
Also add and test for this:

```
<p>class variable defined? {{ class is defined ? 'yes': 'no' }}
```

This is the last thing to fix. I just tested locally. Everything works great (`user` is removed from `attributes`, `{% props` can be used to define defaults, etc) except that if something is NOT defined in `{% props` (i.e. `class`) is IT being made available as a variable, and it should not be. So, in the test, we want to assert that we see:

```
class variable defined? no
```